### PR TITLE
TLS logging for session resumption 1.1

### PR DESCRIPTION
**Pull request by rtkrruvinskiy**
We assume session resumption has occurred if the Client Hello message included
a session id, we have not seen the server certificate, but we have seen a
Change Cipher Spec message from the server.

Previously, these transactions were not logged at all because the server cert
was never seen.

https://redmine.openinfosecfoundation.org/issues/1969

Replaces https://github.com/inliniac/suricata/pull/2423

### CODE REVIEW

**Comment by thus** [comment]
Thanks! I'll take a look at it as soon as I get the chance.

**Comment by thus** [comment]
Prscript:
- PR thus-pcap: https://buildbot.openinfosecfoundation.org/builders/thus-pcap/builds/90
- PR thus: https://buildbot.openinfosecfoundation.org/builders/thus/builds/90

File: src/app-layer-ssl.c [hunk 1]
```diff
@@ -265,6 +265,9 @@ static int TLSDecodeHandshakeHello(SSLState *ssl_state, uint8_t *input,
 
     /* skip session id */
     uint8_t session_id_length = *(input++);
+    if (session_id_length != 0) {
+	ssl_state->flags |= SSL_AL_FLAG_SSL_CLIENT_SESSION_ID;
```
Comment (inliniac) [inline src/app-layer-ssl.c]: style: indent should be 4 spaces

**Comment by rtkrruvinskiy** [comment]
@inliniac Thanks for the review! I'll make those changes.

**Comment by rtkrruvinskiy** [comment]
Replaced by https://github.com/inliniac/suricata/pull/2555

**Pull request closed without merging**